### ISSUE TITLE
Make it understand / ‘.say’ $ / too

### ISSUE THREAD

**Issue by AlexDaniel** [perl6/whateverable#130]
Nowadays evalable allows you to omit ``m:``:
```
<AlexDaniel> say 42
<evalable6> AlexDaniel, rakudo-moar e0e080089: OUTPUT: «42»
```

However, it only works if you start your message with ``say``. A lot of the times people end their messages with ``.say``, like [here](https://irclog.perlgeek.de/perl6/2017-04-16#i_14434656). 

As always, we should find the amount of times it could've mistakenly triggered in the past if the feature was implemented (use quotable for that).

**Comment by AlexDaniel** [comment]
That said, sometimes it triggers when people don't want to:
* https://irclog.perlgeek.de/perl6/2017-07-03#i_14820645

**Comment by AlexDaniel** [comment]
I noticed that every time it was mistriggered, the snippet produced non-zero exit code. Maybe if we add a check for that it will become significantly better.

See also: #53

**Comment by AlexDaniel** [comment]
Also *maybe* reply on [``camelia:``](https://irclog.perlgeek.de/perl6/2017-07-25#i_14918114) also, but exclude real camelia commands like ``help``.

**Comment by AlexDaniel** [comment]
This was properly resolved together with #53, and I think there are enough tests for this feature.
